doc/branching_model.puml

```diff
@@ -3,39 +3,37 @@
 
 == Release ==
 participant "Dev branch" as devbranch order 10 #LightBlue
-participant "RC branch" as rcbranch order 20 #YellowGreen
+participant "Release branch" as rlsbranch order 20 #YellowGreen
 participant "Master branch" as master order 30 #99FF99
 participant "Hotfix branch" as hotbranch order 40 #DarkSalmon
 
-{startrc} devbranch -> rcbranch ++ #YellowGreen: Start release x.y.0
+{startrc} devbranch -> rlsbranch ++ #YellowGreen: Create branch release_x.y.0
 
 
 note left of devbranch
   Evolutions
 end note
 
 
-rcbranch <- devbranch: Merge bug fixes
-rcbranch <- devbranch: Merge more bug fixes
+rlsbranch <- devbranch: Merge bug fixes
+rlsbranch <- devbranch: Merge more bug fixes
 
 
-note right of rcbranch
+note right of rlsbranch
   - Bug fixes
-  - Tag "x.y.0-rc" on RC branch
-  - Push to DockerHub with -rc suffix
```

Comment:
I would still push the rc image to DockerHub. I was thinking just pushing over the docker image tag with a new image if there is a fix merged to keep it simple (and rely on sha256 to distinguish) and not incrementing the tag itself. This is to content people that do not want to build locally in order to try the release candidate. It may be confusing to people otherwise that we release an release candidate, but there is no docker images associated.

Reply:
that was what I was thinking. just push rc but don't spend any effort on individual version tags for rc releases

Reply:
if we keep a rc branch and push to dockerhub for each rc, your original diagram was fine for the most part. can you merge only the .md part and for the diagram let's continue the discussion to the main repo; thanks!